Storage/CompletionOps.swift

```diff
@@ -71,15 +71,17 @@ open class BufferCompletionOp: PerhapsNoOp {
 // move local records to the buffer after upload.
 open class BufferUpdatedCompletionOp: PerhapsNoOp {
     internal let bufferValuesToMoveFromLocal: Set<GUID>
+    internal let deletedValues: Set<GUID>
```

Comment:
Augment the block comment to mention how these are handled — before or after the move? Etc.